spec/api/v2/schemas.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require './spec/api/v2/schemas/security_guides'
+require './spec/api/v2/schemas/metadata'
+
+module Api
+  module V2
+    module Schemas
+      include Metadata
+      include SecurityGuides
```

Comment:
This feels like could be autoloaded in a similar fashion using `descendants` or some other class magic.